latch_cli/snakemake/workflow.py

```diff
@@ -1277,7 +1290,7 @@ def get_fn_code(
                 "inputs": named_list_to_json(job.input),
                 "outputs": named_list_to_json(job.output),
                 "params": {
-                    "keyword": {k: v for k, v in job.params.items()},
+                    "keyword": {k: str(v) for k, v in job.params.items()},
```

Comment:
why this change?

Reply:
This is a minor bug that I found when trying to register a customer's pipeline (separate from latch develop which is why I put it in a separate commit, but I can include it in a separate PR if you prefer that). 

Basically, for snakemake a param value can be a `PosixPath`; however, PosixPath is not json serializable so I convert to string before serializing. This is already handled for input/output in the `named_list_to_json` function.